images/compose/compose.folders.yml

```diff
@@ -51,8 +51,6 @@ services:
       - "./assets/certs:/etc/pulp/certs:z"
       - "../../pulp_storage:/var/lib/pulp:z"
     environment:
-      POSTGRES_SERVICE_PORT: 5432
```

Comment:
I see these are not necessary because they're in the settings.py file. 👍🏻

Reply:
Actually it was only used by the old wait_on_postgres.